manifests/n/Nicotine+/Nicotine+/3.3.4/Nicotine+.Nicotine+.locale.en-US.yaml

```diff
@@ -0,0 +1,32 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Nicotine+.Nicotine+
+PackageVersion: 3.3.4
+PackageLocale: en-US
+Publisher: Nicotine+
+PublisherUrl: https://nicotine-plus.org/
+Author: Avery Terrel
+PackageName: Nicotine+
+PackageUrl: https://nicotine-plus.org/
+License: GPL-3.0
+LicenseUrl: https://github.com/nicotine-plus/nicotine-plus/blob/master/COPYING
+Copyright: Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>

```

Comment:
@averyterrel This isn't the copyright holder of this program, that is who owns the copyright the license document file.

Reply:
You're right that this isn't correct, but upon further inspection, shouldn't it be `Copyright (c) 2001–2024 Nicotine+, Nicotine and PySoulSeek Contributors`? This is what is stated at the very bottom of the README, and seems accurate from what I've `grep`'d

Reply:
Yes, it looks like that would indeed be the best copyright string to use.